### PR TITLE
DOC-1562: RC - Fix broken image in API System Logs example

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-1562/rc/api/examples/audit-system-logs/).

File: content/rc/api/examples/audit-system-logs.md [hunk 1]
```diff
@@ -12,13 +12,16 @@ aliases: /rv/api/how-to/view-auditing-using-system-log/
 ---
 Service logs collect and report actions performed on various entities in your Redis Enterprise Cloud subscription.  These entities include the account itself, users, API Keys, subscriptions, databases, accounts, payment methods, and more. For each entity, various lifecycle events are logged in the system log.
 
-You can view the system service log in the Redis Enterprise Cloud admin console by selecting **Service log** from the console menu. This displays recent service log entries for the current account.
+To view the log, sign into the Redis Cloud [admin console]() and then select Logs from the main menu.
```
Comment (rrelledge) [inline content/rc/api/examples/audit-system-logs.md:15]:
```suggestion
To view the log, sign in to the Redis Cloud [admin console](https://app.redislabs.com/) and then select **Logs** from the main menu.
```

File: content/rc/databases/system-logs.md [hunk 2]
```diff
@@ -10,19 +10,22 @@ aliases: [ "/rv/administration/system-logs/",
...
-{{<image filename="images/rc/icon-database-list-sort-ascending.png" alt="Use the arrows in the list header to sort the list." >}}{{< /image >}} {{<image filename="images/rc/icon-database-list-sort-descending.png" alt="The direction of the arrow corresponds to the direction of the sort." >}}{{< /image >}}    
+
+* Sort the list by a specific field in descending or ascending order. Supported fields include *Time*, *Originator*, *Database name*, and *Activity*.
+
+    {{<image filename="images/rc/icon-database-list-sort-ascending.png" alt="Use the arrows in the list header to sort the list." width="30px">}}{{< /image >}}&nbsp;{{<image filename="images/rc/icon-database-list-sort-descending.png" alt="The direction of the arrow corresponds to the direction of the sort." width="30px">}}{{< /image >}}    
+    
     Select the arrow icon to change the sort order.  You can only sort by one field at a time.
 
-* Use the **Export all** button to export all logs as a CSV file for use in other systems and programs.
+* Use the **Export all** button to export all logs as a comma-separated, variable text (CSV) file for use in other systems and programs.
```
Comment (rrelledge) [inline content/rc/databases/system-logs.md:23]:
```suggestion
* Use the **Export all** button to export all logs as a comma-separated values (CSV) file for use in other systems and programs.
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Looks good to me aside from the admin console link. I left a couple of other small suggestions.

File: content/rc/databases/system-logs.md (hunk 2 above)
Comment (lanceleonard) [reply content/rc/databases/system-logs.md:23]: Fixed.

File: content/rc/api/examples/audit-system-logs.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/api/examples/audit-system-logs.md:15]: Fixed.

**Review by rrelledge — APPROVED** [review]

**Pull request merged**